doc/man/pam.conf-syntax.xml

```diff
@@ -265,7 +265,10 @@
              this action indicates that the return code should be thought
              of as indicative of the module failing. If this module is the
              first in the stack to fail, its status value will be used for
-             that of the whole stack.
+             that of the whole stack.  This is the default action for
+             all return codes, unless overridden via a
+             <emphasis>valueN</emphasis> token or the
+             <emphasis>default</emphasis> token.
```

Comment:
I think this is a little bit confusing given the context. It also repeats the information given below. So I'd just add `This is the default action for all return codes.` here.

Reply:
Done!